extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorControversialTest.java

```diff
@@ -21,7 +23,7 @@
 /**
  * Test for {@link YoutubeStreamLinkHandlerFactory}
  */
-@Ignore("Video is not available in specific countries. Someone else has to generate mocks")
+
```

Comment:
Did you test this against a CI server?

Reply:
It seems to be working: see the log of https://github.com/TeamNewPipe/NewPipeExtractor/runs/2802487877.